sml_builder/templates/cookies.html

```diff
@@ -8,161 +8,249 @@
+                        ],
+                        "trs": [
+                            {
+                                "tds": [
+                                    {
+                                        "value": "<code>licensing_session</code>"
+                                    },
+                                    {
+                                        "value": 'Set to remember information you’ve entered into a form'
+                                    },
+                                    {
+                                        "value": "When you close your browser"
+                                    }
+                                ]
+                            }
+                        ]
+                    })
+                }}
+                <h5 class="ons-u-fs-r--b">Cookie message</h5>
+                <p>You may see a banner when you visit ons.gov.uk inviting you to accept cookies or review your settings. We’ll set cookies so that your computer knows you’ve seen it and not to show it again, and also to store your settings.</p>
+                {{
+                    onsTable({
+                        "ths": [
+                            {
+                                "value": "Name"
+                            },
+                            {
+                                "value": "Purpose"
+                            },
+                            {
+                                "value": "Expires"
+                            }
+                        ],
+                        "trs": [
+                            {
+                                "tds": [
+                                    {
+                                        "value": "<code>ons_cookie_message_displayed</code>"
+                                    },
+                                    {
+                                        "value": 'Lets us know that you’ve seen our cookie message'
+                                    },
+                                    {
+                                        "value": "1 year"
+                                    }
+                                ]
+                            },
+                            {
+                                "tds": [
+                                    {
+                                        "value": "<code>ons_cookie_policy</code>"
+                                    },
+                                    {
+                                        "value": 'Saves your cookie consent settings'
+                                    },
+                                    {
+                                        "value": "1 year"
+                                    }
+                                ]
+                            }
+                        ]
                     })
                 }}
+            {% endcall %}
+            {{
+                onsButton({
+                    "id": 'submit',
```

Comment:
Can you check with @joepercival around these changes - I think there were some accessibility issues around descriptive text etc and you seem to have changed stuff here - I want to check this isn't undoing some of the work Joe has done.

Reply:
On it :)

Reply:
Hi Joe took a look and noted one thing but suspects its to do with the outdated version and thinks his changes will correct this :)